Recorder/insurees-flow-recorder.jmx

```diff
@@ -0,0 +1,3578 @@
+                  <stringProp name="Header.value">pl-PL,pl;q=0.9,en-US;q=0.8,en;q=0.7</stringProp>
+                </elementProp>
+                <elementProp name="Origin" elementType="Header">
+                  <stringProp name="Header.name">Origin</stringProp>
+                  <stringProp name="Header.value">http://${SERVER_NAME}:${SERVER_PORT}</stringProp>
+                </elementProp>
+                <elementProp name="Content-Type" elementType="Header">
+                  <stringProp name="Header.name">Content-Type</stringProp>
+                  <stringProp name="Header.value">application/json</stringProp>
+                </elementProp>
+                <elementProp name="Accept-Encoding" elementType="Header">
+                  <stringProp name="Header.name">Accept-Encoding</stringProp>
+                  <stringProp name="Header.value">gzip, deflate</stringProp>
+                </elementProp>
+                <elementProp name="User-Agent" elementType="Header">
+                  <stringProp name="Header.name">User-Agent</stringProp>
+                  <stringProp name="Header.value">Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/118.0.0.0 Safari/537.36</stringProp>
+                </elementProp>
+                <elementProp name="Accept" elementType="Header">
+                  <stringProp name="Header.name">Accept</stringProp>
+                  <stringProp name="Header.value">*/*</stringProp>
+                </elementProp>
+              </collectionProp>
+            </HeaderManager>
+            <hashTree/>
+          </hashTree>
+          <HTTPSamplerProxy guiclass="GraphQLHTTPSamplerGui" testclass="HTTPSamplerProxy" testname="POST_Create_Insuree_Mutation" enabled="true">
+            <boolProp name="HTTPSampler.postBodyRaw">true</boolProp>
+            <elementProp name="HTTPsampler.Arguments" elementType="Arguments">
+              <collectionProp name="Arguments.arguments">
+                <elementProp name="" elementType="HTTPArgument" enabled="true">
+                  <boolProp name="HTTPArgument.always_encode">false</boolProp>
+                  <stringProp name="Argument.value">{&quot;operationName&quot;:null,&quot;query&quot;:&quot;mutation {\n      createInsuree(\n        input: {\n          clientMutationId: \&quot;9463b13c-b09f-4ba8-8b9e-c7bfd8900fab\&quot;\n          clientMutationLabel: \&quot;Create insuree - testchfid\&quot;\n          \n          chfId: \&quot;testchfid\&quot;\n    lastName: \&quot;test\&quot;\n    otherNames: \&quot;test\&quot;\n    genderId: \&quot;F\&quot;\n    dob: \&quot;2023-10-25\&quot;\n    head: false\n    \n    \n    \n    \n    \n    \n    \n    cardIssued:false\n    \n    \n    \n    familyId: 35\n    \n    status: \&quot;AC\&quot;\n    \n    \n    \n    jsonExt: \&quot;{}\&quot;\n        }\n      ) {\n        clientMutationId\n        internalId\n      }\n    }&quot;}</stringProp>
```

Comment:
family ID and location can be different under different databases. We have think how to parametrize this one.